rules/dnp3-events.rules

```diff
@@ -0,0 +1,17 @@
+# Flooded.
+alert dnp3 any any -> any any (msg:"SURICATA DNP3 Request flood detected"; \
+      app-layer-event:dnp3.flooded; sid:5000000; rev:1;)
```

Comment:
I see that modbus uses 225XXXX, 226XXXX is the app-layer stuff.  Should DNP3 be with the app-layer in 226XXXX? Or something like 227XXXX?

Another thought - should the rules stay in their own file are get merged into app-layer-events.rules?